gravitee-node-cache/pom.xml

```diff
@@ -24,7 +24,7 @@
     <parent>
         <groupId>io.gravitee.node</groupId>
         <artifactId>gravitee-node</artifactId>
-        <version>1.27.1</version>
+        <version>2.0.0-alpha.1</version>
```

Comment:
Do you think we culd take the opportunity to replace it with `${project.version}` ?

Reply:
${project.version} is not usable in parent.
But I used the properties to handle maven versions (like in APIM)